libsolidity/ast/ASTJsonExporter.cpp

```diff
@@ -371,6 +371,7 @@ bool ASTJsonExporter::visit(EnumDefinition const& _node)
 	std::vector<pair<string, Json::Value>> attributes = {
 		make_pair("name", _node.name()),
 		make_pair("nameLocation", sourceLocationToString(_node.nameLocation())),
+		make_pair("documentation", _node.documentation() ? toJson(*_node.documentation()) : Json::nullValue),
```

Comment:
I wonder why don't add the `"documentation"` field for structs. At first I thought it might be because those are not in the ABI but we do that e.g. for modifiers and they aren't in the ABI either.

Reply:
In this PR i'm extending NatSpec for **one** type where it's missing. and if it goes well and in, I can continue on the others. Meanwhile, the reason for not having it all, is that it was originally meant to only document actual public API, not so much as a developer-documentation thing (that I'd like it to be used for _too_). :)